layouts/partials/tabs.html

```diff
@@ -295,22 +295,72 @@ <h3 class="subtitle is-5 is-muted">{{ $subtitle }}</h3>
         </li>
 
         <li id="tab-store" class="cd-tabs__panel text-component">
-        <p>Machine Learning ... .</p>
-
-        <p>
-	This tab is work in progress.  Just parking some content here for now.
-        <a href="https://www.tensorflow.org">Tensorflow</a>
-        framework helps to define and run computations involving tensors that have
-        broad applications in Speech and image recognition, Text-based
-        applications, Time-series analysis and Video detection.
-	
-	Other suggestions for Machine Learning:
-
-        Visualization: Yellowbrick, Eli5
-        ML libraries: XGBoost, LightGBM, CatBoost
-        AI libraries: PyTorch, MXNet
-        
-	</p>
+       	   <div class="grid-container">
+	      <div>
+		      <a href="https://ai.googleblog.com/2016/12/open-sourcing-embedding-projector-tool.html"> <img src="images/content_images/ml_img/pytorch-ml-anim.gif" alt="pytorch figure" align="middle" style="border-radius: 10px"></a> 
```

Comment:
The file name and alt text are confusing - this was made with Tensorflow not PyTorch. Could you change that?

Also, is this okay to include copyright-wise?

Reply:
The image is linked to original one.  I am mentioning source as well so there should be no copyright issue (I think).  I have copied the image for fast loading (as done in other tabs)